src/app/clusters/joint-fabric-administrator-server/joint-fabric-administrator-server.cpp

```diff
@@ -93,7 +93,7 @@ CHIP_ERROR JointFabricAdministratorAttrAccess::Read(const ConcreteReadAttributeP
 
 CHIP_ERROR JointFabricAdministratorAttrAccess::ReadAdministratorFabricIndex(AttributeValueEncoder & aEncoder)
 {
-    return aEncoder.Encode(Server::GetInstance().GetJointFabricDatastore().GetAdministratorFabricIndex());
+    return CHIP_NO_ERROR;
```

Comment:
Why does this function exist, then?

But also, it's weird that we are not getting this from the joint fabric data store.  Why aren't we?

Reply:
JointFabricAdministratorAttrAccess::ReadAdministratorFabricIndex shouldn't exist, it was wrongly introduced by https://github.com/project-chip/connectedhomeip/pull/34637 and then https://github.com/project-chip/connectedhomeip/pull/38842 inexplicable saved the mAdministratorFabricIndex in the Joint Fabric Datastore -by doing this instead of reading the value of mAdministratorFabricIndex@JFAdministratorFabricIndex, its saved value from JF DS implementaton is returned. I opened https://github.com/project-chip/connectedhomeip/issues/39101 to fix this in a separate PR.